euphoria-core/src/main/java/cz/seznam/euphoria/core/client/operator/ReduceByKey.java

```diff
@@ -426,6 +443,11 @@ public boolean isCombinable() {
     return valueExtractor;
   }
 
+  @Override
+  public Dataset<Pair<KEY, OUT>> output(OutputHint... outputHints) {
+    return output();
```

Comment:
This looks odd. Why two methods, when the varargs version can handle the zero argument version as well?